bin/ytdl.js

```diff
@@ -296,6 +297,14 @@ if (opts.infoJson) {
       if (!ext && format.container) {
         output += '.' + format.container;
       }
+
+      // Parses & sanitises output filename for any illegal charcters
+      var parsedOutput = path.posix.parse(output);
+      output = path.format({
```

Comment:
'Guilty Crown - βίος / Bios (Rearranged Medley)'
linux => 'Guilty Crown - βίος / Bios (Rearranged Medley)'
win   => 'Guilty Crown - βίος \ Bios (Rearranged Medley)'

could be fixed by `var parsedOutput = path.parse(sanitizeName(output))`

Reply:
There's no `posix` in the latest version of this file